contrib/completion/bash/docker

```diff
@@ -469,6 +480,7 @@ __docker_tasks() {
 }
 
 # __docker_complete_services_and_tasks applies completion of services and task IDs.
+# shellcheck disable=SC2120
```

Comment:
@jphuynh How would this be done? Can you please give an example for a per-file configuration?